dali/test/dali_test_config.cc

```diff
@@ -33,26 +32,17 @@ std::once_flag noninit_warning;
 
 
 const std::string &dali_extra_path() {
-  if (_dali_extra_path.empty()) {
+  auto ptr = std::getenv("DALI_EXTRA_PATH");
+  if (!ptr) {
     std::call_once(noninit_warning,
-                   []() { std::cerr << "dali_extra_path not initialized. Using current path."; });
+                   []() { std::cerr << "DALI_EXTRA_PATH not initialized."; });
+    _dali_extra_path = "";
```

Comment:
Use something else - perhaps `"."`. Otherwise we'll keep calling std::getenv on each and every call to this function. The other option is to move entire contents of `if (!_dali_extra_path.empty())` body to a function called once.

Reply:
Done